symbolic-ppdb/src/format/sourcelinks.rs

```diff
@@ -124,12 +117,14 @@ mod tests {
 
     #[test]
     fn test_invalid_json() {
-        let mut mappings = SourceLinkMappings::empty();
-        assert!(mappings.add_mappings("").is_err());
-        assert!(mappings.add_mappings("foo").is_err());
-        assert!(mappings.add_mappings("{\"docs\": {\"k\": \"v\"}}").is_err());
+        let mut mappings = SourceLinkMappings::default();
+        assert!(mappings.add_mappings("".as_bytes()).is_err());
+        assert!(mappings.add_mappings("foo".as_bytes()).is_err());
         assert!(mappings
-            .add_mappings("{\"documents\": [\"k\", \"v\"]}")
+            .add_mappings("{\"docs\": {\"k\": \"v\"}}".as_bytes())
+            .is_err());
+        assert!(mappings
+            .add_mappings("{\"documents\": [\"k\", \"v\"]}".as_bytes())
```

Comment:
```suggestion
            .add_mappings(br#"{"documents": ["k", "v"]}"#)
```
You can also use `b"..."` to directly construct a byte slice literal, instead of using `as_bytes`.

Reply:
👍